index.html

```diff
@@ -2189,7 +2189,7 @@ <h2>Service Endpoints</h2>
         <dd>
           <p>
 If a <a>DID document</a> includes a <code>service</code> property, the value
-of the property SHOULD be an <a data-cite="INFRA#ordered-set">ordered set</a>
+of the property MUST be an <a data-cite="INFRA#ordered-set">ordered set</a>
```

Comment:
was this originally a SHOULD because the value could potentially be an ordered set of just URI strings (which dereference to the rest of the data)? Worth checking if this change affects any implementations?

Reply:
I'm not sure we ever had that use case... I may be wrong, but as far as I can remember... service was always supposed to be an array of objects (because you have to give the service an `id`, `type`, and a `serviceEndpoint`)... I don't know of any DID Method that doesn't follow that pattern.